test/controllers/agent/RepresentationControllerSpec.scala

```diff
@@ -57,6 +61,8 @@ class RepresentationControllerSpec extends VoaPropertyLinkingSpec {
 
   behavior of "revokeClient method"
   it should "revoke an agent and display the revoke client page" in {
+    when(revokeClientPropertyPage.apply(any())(any(), any(), any()))
+      .thenReturn(Html("Revoking client <br/> Are you sure you no longer want to act on behalf of"))
```

Comment:
This is not right. This was not here in the original. I'm questioning why you are mocking the page?
Of course the content checks you've added are pointless if you've mocked the page, but you shouldn't need to mock the page if you've mocked your services correctly.

Sorry, this is worse than before.

Reply:
Lets stay professional please

Ok i think I see where you are getting confused. In the play 2.6 version of design systems, you have to inject the html/twirl files, because we are doing so, we need to provide a mock in the tests. Otherwise we just get null pointers. If you have a nicer way around this then please enlighten me

Reply:
"Lets stay professional please" - no idea why you are saying that.

Reply:
Are you saying there is technical reason why we can't render real pages in out unit tests now? Or that we are choosing not to do it?

Reply:
I don't know how you would render the real page, and I don't think we need to, especially if there is no logic to test on the page

Reply:
If you are saying that we can't do it, then the empty html is obviously better than the latest one - which is why I said it is 'worse than before'.

Reply:
> I don't know how you would render the real page, and I don't think we need to, especially if there is no logic to test on the page

Well, we'll end up going in circles. But what if we do want to test the page. Are you saying we can't in 2.6?

Reply:
Im saying, as far as Im aware, you would have to mock the behaviour you want unless you render the real page. I don't know how to do that and there is no point in this case. We can discuss a general case (where we want to test for some logic) separately. It should not need to hold up this PR. Thats really all im trying to say.

Reply:
Agreed that for this page just mock the empty page - I didn't know it was technically difficult. Please revert your last commit and I'll thumb that.

But we need to discuss this as my understanding was that we should be testing pages where there is logic involved.

btw this is a discussion where we can all gain an understanding of what is going on.